### PR TITLE
Rebase dekn#8065

### PR DESCRIPTION
**Pull request by agateaux**
To address https://github.com/plotly/dekn/issues/8065

### CODE REVIEW

File: buildpack.toml [hunk 1]
```diff
@@ -17,301 +17,420 @@ api = "0.7"
...
+    licenses = ["0BSD", "CNRI-Python-GPL-Compatible", "PSF-2.0"]
     name = "Python"
-    purl = "pkg:generic/python@3.7.16?checksum=0cf2da07fa464636755215415909e22eb1d058817af4824bc15af8390d05fb38&download_url=https://www.python.org/ftp/python/3.7.16/Python-3.7.16.tgz"
-    source = "https://www.python.org/ftp/python/3.7.16/Python-3.7.16.tgz"
-    source-checksum = "sha256:0cf2da07fa464636755215415909e22eb1d058817af4824bc15af8390d05fb38"
+    purl = "pkg:generic/python@3.8.19?checksum=c7fa55a36e5c7a19ec37d8f90f60a2197548908c9ac8b31e7c0dbffdd470eeac&download_url=https://www.python.org/ftp/python/3.8.19/Python-3.8.19.tgz"
+    source = "https://www.python.org/ftp/python/3.8.19/Python-3.8.19.tgz"
+    source-checksum = "sha256:c7fa55a36e5c7a19ec37d8f90f60a2197548908c9ac8b31e7c0dbffdd470eeac"
     stacks = ["io.buildpacks.stacks.jammy"]
-    uri = "http://dash-app-resources.plotly-system.svc.cluster.local/packages/kpack-offline-files/python_3.7.16_linux_x64_jammy_c9cecfe2.tgz"
```
Comment (BRONSOLO) [inline buildpack.toml:55]: @tarzzz and/or @agateaux could you please confirm we no longer need to serve these files from `kpack-offline-files`?

Comment (agateaux) [reply buildpack.toml:55]: If you look well, as this diff is not easy to read, offline files are still there, it's just this old version which has been replaced by a newer one like `kpack-offline-files/python_3.8.18_linux_x64_jammy_58676489`.

Python 3.7 has been removed from the upstream.

Comment (BRONSOLO) [reply buildpack.toml:55]: Hm, sorry, I think I'm still missing something : ) I see _some_ references to `kpack-offline-files` still being preserved but I see a lot of changes here that also shows the python package versions being served from `/packages/` and not `/packages/kpack-offline-files`. Is this intended? 

For example: https://github.com/plotly/paketo-buildpacks_cpython/pull/11/files#diff-5f54cb49d0e28117084fad482c0daf89502d4ecb329230071fcf78e56dc939b3R68

Comment (agateaux) [reply buildpack.toml:55]: Ok, this one I need a review. It was added on the upstream, so I also put it there.
Should we remove this type?

**Review by tarzzz — APPROVED** [review]
LGTM!

**Pull request merged**